plugin/evm/vm.go

```diff
@@ -1147,6 +1131,31 @@ func (vm *VM) initBlockBuilding() error {
 		return fmt.Errorf("failed to create atomic tx gossip client: %w", err)
 	}
 
+	atomicTxGossipHandler, err = gossip.NewHandler[GossipAtomicTx, *GossipAtomicTx](
+		vm.ctx.Log,
+		atomicTxGossipClient,
+		vm.mempool,
+		atomicTxGossipHandlerConfig,
+		vm.sdkMetrics,
+	)
+	if err != nil {
+		return err
+	}
+
+	atomicTxGossipHandler = &p2p.ValidatorHandler{
+		ValidatorSet: vm.validators,
+		Handler: &p2p.ThrottlerHandler{
+			Throttler: p2p.NewSlidingWindowThrottler(throttlingPeriod, throttlingLimit),
+			Handler:   atomicTxGossipHandler,
+			Log:       vm.ctx.Log,
+		},
+		Log: vm.ctx.Log,
+	}
```

Comment:
is there an equivalent `New` function for this? If so can we use it (or create if possible)?

Reply:
Argh I missed this one. I'll add this as a follow-up... https://github.com/ava-labs/avalanchego/issues/2482